part_bin_list:
  {
    "part_4": "bin_3_part_4",
    "part_5": "bin_3_part_5",
    "part_6": "bin_3_part_6",
    "part_7": "bin_3_part_7",
    "part_8": "bin_3_part_8",
    "part_9": "bin_3_part_9",
    "part_10": "bin_3_part_10",
    "part_11": "bin_3_part_11",
    "part_12": "bin_3_part_12",
    "part_13": "bin_3_part_13",
    "part_14": "bin_3_part_14",
    "part_15": "bin_3_part_15",
    "part_16": "bin_3_part_16",
    "part_17": "bin_3_part_17",
    "part_18": "bin_3_part_18",
  }
